release/kata-deploy-binaries.sh

```diff
@@ -67,6 +67,22 @@ EOT
 	exit "${return_code}"
 }
 
+#Verify that hub is installed and in case that is not
+# install it to avoid issues when we try to push
+verify_hub() {
+	check_command=$(whereis hub | cut -d':' -f2)
+	# Install hub if is not installed
+	if [ -z ${check_command} ]; then
+		hub_repo="github.com/github/hub"
+		hub_url="https://${hub_repo}"
+		go get ${hub_repo}
```

Comment:
missing `-d` and `|| true`? this can fail if `${GOPATH}/src/${hub_repo}` already exist .. ? :thinking:

Reply:
@devimc changes applied